src/Locale/en/Common.json

```diff
@@ -231,6 +231,11 @@
   "SORT_OPTIONS__-name": "Patient name Z-A",
   "SORT_OPTIONS__bed__name": "Bed No. 1-N",
   "SORT_OPTIONS__-bed__name": "Bed No. N-1",
+  "open_in_full_window": "Open in full window",
+  "minimize_screen":"Minimize",
+  "close_screen":"Close",
+  "send_discussion_message":"Send",
```

Comment:
Can't this be just `Send`? 🤔 Same for the others as its added to the common.

Reply:
> Can't this be just `Send`? 🤔 Same for the others as its added to the common.

please review the changes @bodhish